compiler/prelude/jsmapping.go

```diff
@@ -245,6 +245,9 @@ var $internalize = function(v, t, recv) {
     if (v === null) {
       return $ifaceNil;
     }
+    if (v === undefined) {
+      return new $jsObjectPtr(undefined);
+    }
```

Comment:
This looks good. The JS `undefined` is stored in an interface by wrapping it into `*js.Object`.